### PR TITLE
LSR-1019 Westeros employees hybrid Android

### PR DESCRIPTION
**Pull request by fmicoInfo**
It depends on #438

### CODE REVIEW

File: android/samples/westerosemployees-hybrid/src/main/java/com/liferay/mobile/screens/westerosemployees_hybrid/activities/UserActivity.java [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+	private void bindViews() {
+		TextView userNameTextView = (TextView) findViewById(R.id.liferay_username);
+		userNameTextView.setOnClickListener(this);
+
+		UserPortraitScreenlet userPortraitScreenlet = (UserPortraitScreenlet) findViewById(R.id.userscreenlet_home);
+		userPortraitScreenlet.setOnClickListener(this);
+		userPortraitScreenlet.loadLoggedUserPortrait();
+
+        LastChangeDisplayScreenlet();
```
Comment (nhpatt) [inline android/samples/westerosemployees-hybrid/src/main/java/com/liferay/mobile/screens/westerosemployees_hybrid/activities/UserActivity.java]: tsk, tsk

File: android/samples/westerosemployees-hybrid/src/main/java/com/liferay/mobile/screens/westerosemployees_hybrid/views/BlogsCard.java [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+import com.liferay.mobile.screens.portlet.util.InjectableScript;
+import com.liferay.mobile.screens.westerosemployees_hybrid.R;
+import com.liferay.mobile.screens.westerosemployees_hybrid.utils.CardState;
+
+/**
+ * @author Víctor Galán Grande
+ */
+public class BlogsCard extends CommentsRatingsCard implements PortletDisplayListener {
+
+	PortletDisplayScreenlet portletDisplayScreenlet;
```
Comment (nhpatt) [inline android/samples/westerosemployees-hybrid/src/main/java/com/liferay/mobile/screens/westerosemployees_hybrid/views/BlogsCard.java]: visibility

File: android/samples/westerosemployees-hybrid/src/main/java/com/liferay/mobile/screens/westerosemployees_hybrid/views/DocsCard.java [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+import com.liferay.mobile.screens.portlet.PortletDisplayListener;
+import com.liferay.mobile.screens.portlet.PortletDisplayScreenlet;
+import com.liferay.mobile.screens.portlet.util.InjectableScript;
+import com.liferay.mobile.screens.westerosemployees_hybrid.R;
+import com.liferay.mobile.screens.westerosemployees_hybrid.utils.CardState;
+
+/**
+ * @author Víctor Galán Grande
+ */
+public class DocsCard extends CommentsRatingsCard implements PortletDisplayListener {
```
Comment (nhpatt) [inline android/samples/westerosemployees-hybrid/src/main/java/com/liferay/mobile/screens/westerosemployees_hybrid/views/DocsCard.java]: Have you talked about using westeros as a dependency to not have to copy all these classes over again?

File: android/samples/westerosemployees-hybrid/src/main/res/layout/card_comments_red.xml [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+					<ImageView
+						android:layout_alignParentEnd="true"
+						android:layout_alignParentRight="true"
+						android:src="@drawable/icon_down"
+						android:tag="@string/arrow_tag"
+						android:visibility="invisible"
+						style="@style/westeros_arrow"
+						/>
+
+                    <TextView
```
Comment (nhpatt) [inline android/samples/westerosemployees-hybrid/src/main/res/layout/card_comments_red.xml]: Is this formatted? maybe it is and github doesn't show it right

File: android/samples/westerosemployees-hybrid/src/main/res/raw/blog_portlet_js.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
+function modifyItem() {
+
+    var etqA = document.getElementsByTagName("a");
+        var etqImg = document.getElementsByTagName("img");
```
Comment (nhpatt) [inline android/samples/westerosemployees-hybrid/src/main/res/raw/blog_portlet_js.js]: mmm...

File: android/samples/westerosemployees-hybrid/src/main/res/raw/blog_portlet_js.js [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+        for(var i = 0; i < etqA.length; i++) {
+            etqA[i].removeAttribute("href");
```
Comment (nhpatt) [inline android/samples/westerosemployees-hybrid/src/main/res/raw/blog_portlet_js.js]: dont use so short variable names :)

File: android/samples/westerosemployees-hybrid/src/main/res/raw/blog_portlet_js.js [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+            if(etqImg[i].id != "img-header"){
+                imageToDiv(etqImg[i]);
+            }
+        }
+}
+
+modifyItem();
+
+function cleanRatingText(rating){
+    rating.innerHTML = rating.textContent.replace("Average (", "").replace(")", "");
```
Comment (nhpatt) [inline android/samples/westerosemployees-hybrid/src/main/res/raw/blog_portlet_js.js]: strings with ''

File: android/samples/westerosemployees-hybrid/src/main/res/raw/blogs_portlet_js.js [hunk 8]
```diff
@@ -0,0 +1,27 @@
+function modifyItem() {
+
+    var items = document.querySelectorAll(".item-blog");
+
+    for(var i = 0; i < items.length; i++) {
+        items[i].style = null;
+        addClick(items[i]);
+        if(i != items.length - 1){
```
Comment (nhpatt) [inline android/samples/westerosemployees-hybrid/src/main/res/raw/blogs_portlet_js.js]: js format... if (

File: android/settings.gradle [hunk 9]
```diff
@@ -10,7 +10,7 @@ include 'liferay-westeros-viewset'
 project(':liferay-westeros-viewset').projectDir = new File(settingsDir, '/viewsets/westeros')
 project(':liferay-westeros-viewset').name = 'liferay-westeros-viewset'
 
-include ':addbookmarkscreenlet'
+include ':addbookmarkscreenlet', ':westerosemployees-hybrid'
```
Comment (nhpatt) [inline android/settings.gradle]: pass android lint to see it there are issues in your module, feel free to ask any doubts whatsoever to me or victor :)

File: android/library/src/main/assets/gallery.js [hunk 10]
```diff
@@ -1,17 +1,13 @@
 function addClickToCards() {
-	var cards = document.getElementsByClassName('card');
+	var cards = document.getElementsByClassName("card");
```
Comment (nhpatt) [inline android/library/src/main/assets/gallery.js]: formatting

File: android/library/src/main/assets/gallery.js [hunk 11]
```diff
@@ -1,17 +1,13 @@
...
     var allImgSrc = "";
```
Comment (nhpatt) [inline android/library/src/main/assets/gallery.js]: strings with ''

File: android/samples/westerosemployees-hybrid/src/main/AndroidManifest.xml [hunk 12]
```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.liferay.mobile.screens.westerosemployees_hybrid">
+
+    <uses-permission android:name="android.permission.INTERNET"/>
```
Comment (nhpatt) [inline android/samples/westerosemployees-hybrid/src/main/AndroidManifest.xml]: pretty sure this is not formatted. The idea configuration is uploaded here, import it https://github.com/liferay/liferay-screens/blob/develop/android/android-studio-inspections.xml and auto format

File: android/samples/westerosemployees-hybrid/src/main/java/mx/ferreyra/callnumber/CFCallNumber.java [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+import java.util.List;
+
+import android.Manifest;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.telephony.TelephonyManager;
+import android.content.pm.PackageManager;
+
+public class CFCallNumber extends CordovaPlugin {
```
Comment (nhpatt) [inline android/samples/westerosemployees-hybrid/src/main/java/mx/ferreyra/callnumber/CFCallNumber.java]: If you copy this class, at least format it...

**Comment by nhpatt** [comment]
Can you rebase?

**Comment by fmicoInfo** [comment]
@nhpatt rebase done :)

**Comment by nhpatt** [comment]
Merged here: https://github.com/liferay/liferay-screens/pull/455

Why the css and js files end with _css.css ???

**Pull request merged**